src/Directory.Build.props

```
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <VersionPrefix>4.5.0-preview</VersionPrefix>
+    <VersionPrefix>4.5.1-preview</VersionPrefix>
```

```suggestion
    <VersionPrefix>4.5.0-preview</VersionPrefix>
```

I was wrong. Version was already bumped from 4.4 to 4.5 in previous PR.